rskj-core/src/main/java/co/rsk/peg/BridgeSerializationUtils.java

```diff
@@ -132,7 +116,23 @@ private BridgeSerializationUtils(){}
         return map;
     }
 
-    public static List<UTXO> deserializeList(byte[] data) throws IOException {
+    public static byte[] serializeUTXOList(List<UTXO> list) throws IOException {
+        int nutxos = list.size();
+
+        byte[][] bytes = new byte[nutxos][];
+        int n = 0;
+
+        for (UTXO utxo : list) {
+            ByteArrayOutputStream ostream = new ByteArrayOutputStream();
+            utxo.serializeToStream(ostream);
+            ostream.close();
```

Comment:
if `serializeToStream` throws an exception, the *ostream* never gets closed. You can use the _try with resources_ syntax here like this:
```java
try(ByteArrayOutputStream ostream = new ByteArrayOutputStream()) {
    utxo.serializeToStream(ostream);
    bytes[n++] = RLP.encodeElement(ostream.toByteArray());
}
```

Reply:
java.io.ByteArrayOutputStream.close() is an empty method

From the impl
```Java
    public void close() throws IOException {
    }
```

Reply:
What @donequis said. And plus, this method didn't actually change, only its name.

Reply:
That doesn't necessarily mean we don't actually need to call the `close()`, just saying its been there for a while.